inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkService.java

```diff
@@ -63,16 +70,41 @@
     private final Integer parallelism;
     private final String savepointDirectory;
 
-    public FlinkService() throws IOException {
+    public FlinkService(String endpoint) throws IOException {
         FlinkConfiguration flinkConfiguration = new FlinkConfiguration();
         flinkConfig = flinkConfiguration.getFlinkConfig();
-        address = flinkConfig.getAddress();
-        port = flinkConfig.getPort();
         jobManagerPort = flinkConfig.getJobManagerPort();
         parallelism = flinkConfig.getParallelism();
         savepointDirectory = flinkConfig.getSavepointDirectory();
+        if (endpoint.length() == 0) {
+            address = flinkConfig.getAddress();
+            port = flinkConfig.getPort();
+        } else {
+            address = translateFromEndpont(endpoint).get("address");
+            port = Integer.valueOf(translateFromEndpont(endpoint).get("port"));
```

Comment:
`port` use string data type directly. Not to transform int data type. It is redundancy.